docs/docusaurus/versioned_docs/version-1.7.0/general/cloudstrapper_install.md

```diff
@@ -108,7 +108,7 @@ varFirstInstall: "false"
     - varBuildType: Using either ‘community’ or ‘custom’ binaries
     - varFirstInstall: Indicating if this is the first install of any kind of Magma or not, to skip some of the default, shared roles created
 
-- First change your directory to ~/magma-dev/magma/experimental/cloudstrapper/playbooks. Next, run the playbook to setup Orchestrator the deployment:
+- First change your directory to `~/magma-dev/magma/experimental/cloudstrapper/playbooks`. Next, run the playbook to setup the Orchestrator deployment:
```

Comment:
```suggestion
- First change your directory to `~/magma-dev/magma/experimental/cloudstrapper/playbooks`. Next, run the playbook to set up the Orchestrator deployment:
```

Reply:
In case you weren't aware, it looks like you reverted my suggested changes while force pushing

Reply:
FIxed it back